pkg/query/query.go

```diff
@@ -84,3 +85,7 @@ func (m *Cache) GetQuery(lang core.LanguageId, name string) (*sitter.Query, bool
 	}
 	return nil, false
 }
+
+func Join(queries ...string) string {
```

Comment:
The looks unused?

Reply:
Will remove. I've changed approaches in places that I had been joining multiple queries together.